app/assets/stylesheets/application.scss

```diff
@@ -18,6 +18,8 @@ body {
   height: 100%;
 }
 
+.clearfix {clear:both;}
```

Comment:
I still believe we should find the source of this issue instead of putting in the above.  Unfortunately I don't have any suggestions at the moment.